src/app/core/services/rundown-state.service.ts

```diff
@@ -125,15 +132,17 @@ export class RundownStateService implements OnDestroy {
       return
     }
 
-    part.insertAdLibPiece(event.adLibPiece)
+    // TODO: Out-commented due to current changes in AdLib API. Wait until that is somewhat stable.
+    //part.insertAdLibPiece(event.adLibPiece)
```

Comment:
I think you can safely remove these. Same goes for the similarly outcommented adlib related line further down.

Reply:
It has been removed.